src/widgets/settingspages/ExternalToolsPage.cpp

```diff
@@ -14,6 +14,15 @@
 
 namespace chatterino {
 
+const char *getBinaryName()
+{
+#ifdef _WIN32
+    return "streamlink.exe";
+#else
+    return "streamlink";
+#endif
+}
```

Comment:
This should be in an anonymous namespace (`namespace { }`) above. Or, even better imo: create a `constexpr QStringView STREAMLINK_BINARY_NAME = u"abc";`

Reply:
@Nerixyz Could you give your opinion on how it's done here [`4e19758` (#5076)](https://github.com/Chatterino/chatterino2/pull/5076/commits/4e19758c047e368690a1d17ce05c2b895427da3b) ?